lte/gateway/c/oai/common/amf_default_values.h

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Please use the Magma copyright for any new additions, as seen here https://github.com/magma/magma/blob/master/lte/gateway/c/session_manager/AAAClient.cpp